app/assets/javascripts/about.js.coffee

```diff
@@ -1,3 +1,9 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+object = {
+  spacing: true,
+  spacing: true,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Duplicate key defined in object or class](https://www.codacy.com/app/actcat/hogehoge/file/2965814450/issues/source?bid=3079895&fileBranchId=3428823#l7)